test/test_dgmulti_2d.jl

```diff
@@ -12,6 +12,47 @@ outdir = "out"
 isdir(outdir) && rm(outdir, recursive=true)
 
 @testset "DGMulti 2D" begin
+
+  @trixi_testset "SemidiscretizationHyperbolicParabolic" begin
```

Comment:
This should probably be tested elsewhere?

Reply:
Agreed, I'll move it into its own testset